packages/core/src/generators/react.ts

```diff
@@ -51,10 +51,18 @@ export interface ToReactOptions extends BaseTranspilerOptions {
   stateType?: 'useState' | 'mobx' | 'valtio' | 'solid' | 'builder';
   format?: 'lite' | 'safe';
   type?: 'dom' | 'native';
+  preact?: boolean;
```

Comment:
wouldn't it make more sense to add `preact` to `type`?

```suggestion
  type?: 'react' | 'react-native' | 'preact';
```

Reply:
That was my initial impulse but actually it would’ve made the code more complicated and harder to maintain given preact is almost identical to dom with just a couple small exceptions regarding imports

Reply:
Also fwiw you can use preact with react native, so they aren’t mutually exclusive options